erpnext/stock/report/item_prices/item_prices.py

```diff
@@ -82,33 +82,34 @@ def get_last_purchase_rate():
 
 	query = """select * from (select
 					result.item_code,
-					result.base_rate
+					result.base_net_rate
 					from (
 						(select
 							po_item.item_code,
-							po_item.item_name,
-							po.transaction_date as posting_date,
-							po_item.base_price_list_rate,
-							po_item.discount_percentage,
-							po_item.base_rate
+							po_item.base_net_rate
 						from `tabPurchase Order` po, `tabPurchase Order Item` po_item
-						where po.name = po_item.parent and po.docstatus = 1)
+						where po.name = po_item.parent and po.docstatus = 1
+						order by po.transaction_date desc, po.name desc limit 1)
```

Comment:
Will this line really work?
The idea of this function is to create a `item_last_purchase_rate_map = {}`
This line would return only a single row from all the Purchase Order Items matching their Parent POs.

As my understanding of MySQL is limited.
I ran this in my environment - and it indeed churns up a single row because of limit 1. I remove that, all the item lists start coming up. Only when all Purchase Order Items are as output, will it match with their respective PR and PIs.
If that remains, the map will never be created.

Reply:
Yep. You are correct. The set of data I tested on generated same result so I didn't catch the problem. Will modify the query to fetch one po/pr/pi transaction for every stock item.